volumes/volume-manage.html.markerb

```diff
@@ -176,19 +178,34 @@ For options, refer to the [`fly machine clone` docs](/docs/flyctl/machine-clone/
 ## Destroy a volume
 
 <div class="warning icon">
-<b>Warning:</b> When you destroy a volume, you permanently delete all its data.
+<b>Warning:</b> When you destroy a volume, you permanently delete all its data. There is no undo. Make sure you've backed up any data you want to keep.
 </div>
 
-1. Run `fly volumes list` and copy the ID of volume to destroy.
+1. List volumes and find the ID for the one you want to destroy:
 
-2. Destroy the volume:
+```cmd
+fly volumes list
+```
+
+2. Destroy the volume using its ID:
 
 ```cmd
 fly volumes destroy <volume id>
 ```
 
 For options, refer to the [`fly volumes destroy` docs](/docs/flyctl/volumes-destroy/) or run `fly volumes destroy --help`.
 
+**Volumes attached to Machines**
```

Comment:
I've modified your suggestion a bit, let me know if this works!

Reply:
Looks perfect, thanks!